.github/workflows/unit-test.yml

```diff
@@ -103,26 +103,25 @@ jobs:
     runs-on: ubuntu-20.04
     timeout-minutes: 15
     steps:
-      - name: Check out the repo
-        uses: actions/checkout@v3
-
-      - name: Set up GO (with caching)
-        uses: magnetikonline/action-golang-cache@v4
-        with:
-          go-version-file: .go-version
-
       - name: Check out the repo
         uses: actions/checkout@v3
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           repository: ${{ (github.event.pull_request_target || github.event.pull_request).head.repo.full_name }}
           ref: ${{ (github.event.pull_request_target || github.event.pull_request).head.sha }}
 
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: .go-version
+          cache: false
```

Comment:
Seems like this would remove the caching, no?

Reply:
Caching is supposed to be handled in https://github.com/golangci/golangci-lint-action